nntrainer/tensor/float_tensor.cpp

```diff
@@ -168,18 +168,71 @@ void FloatTensor::setZero() {
   }
 }
 
-/// @todo support additional initializer
+void FloatTensor::setRandNormal(float mean, float std) {
```

Comment:
I understand that the variable name "std" is being used for and it was previously used in "tensor.cpp" and "tensor.h", but using the variable name "std" in C++ code does not seem like a good way.

Would it be possible to change all the existing variable names along with the new one we are writing now?
for example in tensorflow (keras) they use like below

tf.keras.initializers.RandomNormal

```
tf.keras.initializers.RandomNormal(
    mean=0.0, stddev=0.05, seed=None
)
```

or in numpy they use

```
random.normal(loc=0.0, scale=1.0, size=None)
```

We acknowledge that PyTorch uses the same name as us, but since it is based on Python, we believe it to be different.

Reply:
100% agree. will fix it

Reply:
fixed!